libfwupdplugin/fu-progress.h

```diff
@@ -64,6 +64,15 @@ typedef guint64 FuProgressFlags;
  */
 #define FU_PROGRESS_FLAG_NO_PROFILE (1ull << 1)
 
+/**
+ * FU_PROGRESS_FLAG_CHILD_FINISHED:
+ *
+ * The child did not complete all the required steps, and that is expected.
```

Comment:
this english reads funny.  I think it should just be "The child completed all expected steps"

Reply:
Ack, thanks.